tests/test/standard-flavor/8.0/available-packages.py

```diff
@@ -0,0 +1,50 @@
+#!/usr/bin/env python3
+
+
+from exasol_python_test_framework import udf
+from exasol_python_test_framework.exatest.testcase import useData
+
+
+class AvailablePythonPackages(udf.TestCase):
+    def setUp(self): 
+        self.query('create schema available_packages', ignore_errors=True)
+
+    data = [
+            ("pybase64",),
+            ("simdjson",),
+            ("numba",),
+            ("pyarrow",),
+            ("bitarray",),
+            ("pybloomfilter",),
+            ("bitsets",),
+            ("yaml",),
+        ]
+
+    @useData(data)
+    def test_package_import(self, pkg, fail=False, alternative=None):
+        sql=udf.fixindent('''
+            CREATE OR REPLACE PYTHON3 SCALAR SCRIPT available_packages.test_import_of_package() returns int AS
+            
+            import %s
+            def run(ctx): return 1
+            /
+            ''' % (pkg))
+        print(sql)
```

Comment:
is the print(..)  debug code?

Reply:
I add this to the tests, to make the analysis a bit easier, so here no